examples/SortExample.js

```diff
@@ -99,7 +99,7 @@ class SortExample extends React.Component {
       if (valueA < valueB) {
         sortVal = -1;
       }
-      if (sortVal !== 0 && sortDir === SortTypes.ASC) {
+      if (sortVal !== 0 && sortDir === SortTypes.DESC) {
```

Comment:
#312 
Default comparison  `function(a, b)` should return `a - b`;
And since the default comparison is ascending, we should only invert the value when DESCENDING is specified.
Also arrows aren't reversed (which was the intent of the original commit [7ddb0cd].

Reply:
Great, thanks for digging into this!